user/super/com/google/gwt/emul/java/text/Normalizer.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2024 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+
+package java.text;
+
+import javaemul.internal.JsUtils;
+import jsinterop.annotations.JsType;
+
+/**
+ * Emulation of {@code java.text.Normalizer}.
```

Comment:
Note that this is circular - maybe a link to javadoc instead? e.g. https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/text/Normalizer.html